docs/lab/01.md

```diff
@@ -2,4 +2,488 @@
+
+<div align="center">
+![Power symbol](images/power_symbol.png)   
+Add power symbol
+</div>
+
+Pick $$GND$$ and place it inside the sheet borders. Connect the power signal to the $$GND$$ pins of the microcontroller. To connect, click on `Add a wire` or use the shortcut `W`.
+
+
+If we take a look in the development board [datasheet](https://datasheets.raspberrypi.com/pico/pico-datasheet.pdf) at the power chain section, we can see that the power supplied of the USB connector, $$V_{BUS}$$, which is at $$5V$$ as standard, is regulated by a voltage regulator at $$3V3$$. Place a power symbol for $$V_{BUS}$$, $$3V3$$ and a $$5V$$ power symbol for $$V_{SYS}$$.
+
+:::warning
+Even if $$V_{SYS}$$ and $$V_{BUS}$$ have the same value, between them there is a protection diode for reverse current that protects both the microcontroller and the USB port of your PC. Be careful to not connect this 2 pins as it will cancel the protection diode.
+:::
+
+<div align="center">
+![Power chain](images/pico_powerchain.png)   
+Power Chain Raspberry Pi Pico
+</div>
+
+Complete the schematic by placing an LED and a resistor at any given GPIO pin.
+
+<div align="center">
+![Pico Led Schematic](images/led_schematic.png)   
+LED Schematic
+</div>
+
+Save the file and open the PCB editor. You can use the button on the top bar of the Symbol editor.
+
+Use the highlighted button to update the PCB with the changes made in Schematic editor.
+
+<div align="center">
+![Update PCB](images/update_pcb_1.png)   
+Update PCB
+</div>
+
+You will get an error because for the resistor $$R1$$ and LED $$D1$$ there are no footprints assigned. 
+
+<div align="center">
+![Footprint error ](images/footprint_error.png)   
+Footprint error 
+</div>
+
+Go back to the Schematic and assign footprints for resistor $$R1$$ and LED $$D1$$ like in the setup [tutorial](../tutorials/kicad.md). You can pick any footprint, or just paste these on footprint field in symbol properties.
+- resistor: Resistor_THT:R_Axial_DIN0204_L3.6mm_D1.6mm_P2.54mm_Vertical
+- LED: LED_THT:LED_D3.0mm
+
+Go back to the PCB editor and try to update again.
+
+<div align="center">
+![Update PCB](images/update_pcb_2.png)   
+Update PCB
+</div>
+
+Connect the components using the `Route tracks` button on the right or the shortcut `X`. The blue lines will guide you to connect the right pins.
+
+<div align="center">
+![Route tracks](images/route_tracks.png)   
+Route tracks
+</div>
+
+KiCad offers a 3D Viewer to have a preview of how the PCB will look. Click on `View` -> `3D Viewer`
+
+<div align="center">
+![3D View](images/3D_view.png)   
+3D View
+</div>
+
+:::info
+For a better understanding, please read [Getting Started in KiCad](https://docs.kicad.org/7.0/en/getting_started_in_kicad/getting_started_in_kicad.html) tutorial.
+:::
+
+## Exercises
+
+1. Complete the given exercise sheet.(**3p**)
```

Comment:
Add a download link here to the pdf, uou can add it in assets.

Reply:
I want to make different versions, print them and give it to the students